tools/importer/tfm_importer.json

```diff
@@ -64,12 +68,12 @@
     }
   ],
   "commit_sha": [
-    "fb068d2cb4e89cacf0e9f413075bb4b211f1484f",
-    "5d41a2aeae71b13f2763bea4e55899646291e0eb",
-    "9c1e080e39adc7211c8c2c12cd652da3dc124299",
-    "78ed87028718b1b926d847ff6fc2f91d44e53d6d",
-    "280715f9b74ab29459d81edaf02b39e7a6acb13c",
-    "ea81bf91c90ae23dd9de012bfd7498613be00601",
-    "5342015bb12a486a1c563175a8a7129f0737c925"
+    "11bff3f3cbfbd3e2c284e884d0066531e6b47d7e",
```

Comment:
Don't know what these are - should they be committed?

Reply:
These are commit hashes of patches present in the Mbed OS repo (in this very PR, in fact!), necessary to complete the import and integration of TF-M into Mbed OS. Yes, they are needed and should be committed.